services/playlists/src/services/playlistService.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { Playlists } = require('../db/models/Playlists');
+const { v4: uuid } = require('uuid');
+const { errorMessages } = require('../config');
+
+class PlaylistService {
+  async getPlaylists() {
+    return Playlists.findAll({ order: ['id'] });;
+  }
+
+  async getPlaylistById(id) {
+    const playlist = await Playlists.findOne({ where: { id } });
+
+    if (!playlist) throw new Error(errorMessages.PLAYLIST_NOT_EXISTS_ID);
+
+    return playlist;
+  }
+
+  async createPlaylist(playlistData) {
+    const playlistID = uuid();
+    playlistData.id = playlistID;
+
+    return Playlists.create(playlistData);
```

Comment:
It is a bad practice to modify function params. Instead of it, you should create a new object and put in it data using spread operator, and then modify it as you need.
```suggestion
    const newPlaylist = { ...playlistData, id: uuid() };
    
    return Playlists.create(newPlaylist);
```